server/src/main/java/org/spine3/server/aggregate/AggregateRoot.java

```diff
@@ -40,7 +40,8 @@
 
     /** The map from a part class to a repository which manages corresponding aggregate part */
     private static final Map<Class<? extends Message>,
-            AggregatePartRepository<?, ?, ?>> partsAccess = Maps.newConcurrentMap();
+                             AggregatePartRepository<?, ?, ?>> partsAccess =
+                                                               Maps.newConcurrentMap();
 
     /** The bounded context to which the aggregate belongs. */
     private final BoundedContext boundedContext;
```

Comment:
Please make `getBoundedContext()` private too. We want to limit the ability for the end developer to do “dirty tricks” by accessing B.C. directly.